static/sass/_pattern_card.scss

```diff
@@ -1,44 +1,80 @@
 @mixin insights-p-card {
 
-  %p-card__header {
-    color: $color-x-light;
-    margin: -21px -21px 21px -21px;
-    padding: 1rem;
-  }
+  .p-card__category {
+    background: url('#{$assets-path}ed42aefa-icon-resource-hub-icon-document.png') left center no-repeat;
+    color: $color-mid-dark;
+    font-size: $sp-medium;
+    line-height: 1.5;
+    padding: 0 0 0 $sp-large;
+    text-transform: uppercase;
 
-  .p-card__header--desktop {
-    @extend %p-card__header;
-    background-color: #749F8D;
-  }
+    > a:link,
+    > a:visited {
+      color: $color-mid-dark;
+      text-decoration: none;
+    }
 
-  .p-card__header--canonical-announcements {
-    @extend %p-card__header;
-    background-color: #FF8936 ;
+    > a:hover,
+    > a:active {
+      color: $color-brand;
+      text-decoration: underline;
+    }
   }
 
-  .p-card__header--cloud-and-server {
-    @extend %p-card__header;
-    background-color: #48929B;
+  %resources-card-header {
+    border-radius: 2px;
+    padding: $sp-small $sp-medium;
   }
 
-  .p-card__header--internet-of-things {
-    @extend %p-card__header;
-    background-color: #A87CA0;
-  }
+  .p-card--resources {
+    @extend .p-card--highlighted;
+    display: flex;
```

Comment:
The cookie-policy css is interfering with this, and turning it into a block instead of a flex. For now you can probably just add `!important` and we can figure out why later.